roadside-forms-frontend/frontend_web_app/src/components/CommonForm/registeredOwnerInfo.js

```diff
@@ -1,89 +1,177 @@
-import { useEffect } from 'react';
-import PropTypes from 'prop-types';
-import { SearchableSelect } from "../common/Select/SearchableSelect"
-import { useFormikContext } from 'formik';
-import "./commonForm.scss"
+import { Checkbox } from "../common/Checkbox/checkbox";
+import { Input } from "../common/Input/Input";
+import Button from "react-bootstrap/Button";
+import { PhoneField } from "../common/Input/phoneField";
+import { useEffect } from "react";
+import PropTypes from "prop-types";
+import { SearchableSelect } from "../common/Select/SearchableSelect";
+import { useFormikContext } from "formik";
+import "./commonForm.scss";
+import { Row, Col } from "react-bootstrap";
+import { DateOfBirthField } from "../common/DateField/dateOfBirthField";
 
 export const RegisteredOwnerInfo = (props) => {
-    const { provinces } = props;
-    const { values, setValues } = useFormikContext();
-    const ownedByCorp = values['owned-by-corp'];
+  const { provinces } = props;
+  const { values, setValues } = useFormikContext();
+  const ownedByCorp = values["owned_by_corp"];
 
+  useEffect(() => {
+    // If user autofill owner info from driver info and then click owned by corp checkbox, this logic will remove autofilled owner info
+    if (
+      ownedByCorp &&
+      (values["regist_owner_last_name"] !== "" ||
+        values["regist_owner_first_name"] !== "")
+    ) {
+      const updatedValues = {
+        ...values,
+        regist_owner_last_name: "",
+        regist_owner_first_name: "",
+      };
 
-    useEffect(() => {
-        // If user autofill owner info from driver info and then click owned by corp checkbox, this logic will remove autofilled owner info
-        if (ownedByCorp && (values['regist_owner_last_name'] !== '' || values['regist_owner_first_name'] !== '')) {
-          const updatedValues = {
-            ...values,
-            'regist_owner_last_name': '',
-            'regist_owner_first_name': ''
-          };
-      
-          setValues(updatedValues);
-        }
-      }, [ownedByCorp, setValues, values]);
+      setValues(updatedValues);
+    }
+  }, [ownedByCorp, setValues, values]);
 
-    const handlePopulateFields = () => {
-        // Get the driver information and populate registered owner fields when prefill button is clicked
-        const driverInfo = {
-          "regist_owner_last_name": ownedByCorp ?  '' : values['driver_last_name'] ,
-          "regist_owner_first_name": ownedByCorp ? '': values['driver_given_name'],
-          "regist_owner_address": values['driver_address'],
-          "regist_owner_phone": values['driver_phone'],
-          "regist_owner_city": values['driver_city'],
-          "regist_owner_prov_state":  values['driver_prov_state'],
-          "regist_owner_postal":  values['driver_postal'],
-        };
-        setValues({
-            ...values,
-            ...driverInfo,
-          });
-        };
+  // TODO: Clear driver first + last name + DOB when corporate ownership checked
```

Comment:
just checking again if this is still needed

Reply:
removed